src/components/sidebar/Sidebar.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+    <vs-sidebar
+        square
+        v-model="active"
+        v-click-outside="closeSidebar"
+        :open="desktopMediaQuerys ? desktopMediaQuerys : isSidebarShown"
+    >
+        <vs-sidebar-group>
+            <template #header>
+                <vs-sidebar-item arrow>
+                    {{ $t('sections.intro') }}
+                </vs-sidebar-item>
+            </template>
+
+            <vs-sidebar-item id="Instagram"> Instagram </vs-sidebar-item>
+            <vs-sidebar-item id="twitter"> Twitter </vs-sidebar-item>
+            <vs-sidebar-item id="Facebook"> Facebook </vs-sidebar-item>
+        </vs-sidebar-group>
+        <vs-sidebar-group>
+            <template #header>
+                <vs-sidebar-item arrow>
+                    {{ $t('sections.api') }}
+                </vs-sidebar-item>
+            </template>
+
+            <vs-sidebar-item id="github"> Github </vs-sidebar-item>
+            <vs-sidebar-item id="codepen"> Codepen </vs-sidebar-item>
+            <vs-sidebar-item id="discord"> Discord </vs-sidebar-item>
+            <vs-sidebar-item id="Javascript"> Javascript </vs-sidebar-item>
+            <vs-sidebar-item id="git"> Git </vs-sidebar-item>
+        </vs-sidebar-group>
+        <vs-sidebar-group>
+            <template #header>
+                <vs-sidebar-item arrow>{{
+                    $t('sections.expressjs')
+                }}</vs-sidebar-item>
+            </template>
+
+            <vs-sidebar-item id="github"> Github </vs-sidebar-item>
+            <vs-sidebar-item id="codepen"> Codepen </vs-sidebar-item>
+            <vs-sidebar-item id="discord"> Discord </vs-sidebar-item>
+            <vs-sidebar-item id="Javascript"> Javascript </vs-sidebar-item>
+            <vs-sidebar-item id="git"> Git </vs-sidebar-item>
+        </vs-sidebar-group>
+        <vs-sidebar-group>
+            <template #header>
+                <vs-sidebar-item arrow>{{
+                    $t('sections.mongoosejs')
+                }}</vs-sidebar-item>
+            </template>
```

Comment:
@PVillasana Apply here manual format to fix the structure of the vs-sidebar-item

Reply:
@PVillasana I think that you have not translation any words because you will not use these words in the project, but if you want use (Github, Codepen, Discord etc..) the words, please apply the tanslation and fix the name of the ID, because in any case you use Capital letter y the first letter (PascalCase).

Reply:
@gonzalo-plaza those are dummy data, there will be removed.

Reply:
@PVillasana Nice, I resolve the conversation.